### DISCUSSION TITLE
Missing some files in branche 'next' .

### DISCUSSION THREAD

**Discussion by chelal233** [protobuf-c/protobuf-c#588] [Q&A] [answered]
Hello,

Files "protobuf-c.pb.cc" and "protobuf-c.pb.h" is required in commits #466, but they are not added to github.

Can someone take them to github?

Thank you

**Comment by lipnitsk** [comment, marked as answer]
No, those files are auto-generated by protoc and only used by tests or when building the compiler, not the runtime itself. They will never be added to the repository. What problem are you facing?
